apps/angular/src/app/services/anime.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+import { map } from 'rxjs/operators';
+
+import { Anime } from '../entities/anime';
+
+import { AllAnimeResponseDto } from '../dto/allAnimeResponse.dto';
+
+/** Anime service. */
+@Injectable({
+	providedIn: 'root',
+})
+export class AnimeService {
+
+	private apiUrl = 'https://api.camp-js.saritasa.rocks';
+
+	private allAnimeEndpoint = 'api/v1/anime/anime/';
```

Comment:
I believe we don't want to mutate this field
```suggestion
	private readonly allAnimeEndpoint = 'api/v1/anime/anime/';
```